### PR TITLE
Related Work

### PR DESCRIPTION
**Pull request by ClemensLinnhoff**
**Reference to a related issue in the repository**
#15 

**Add a description**
Added list of related work and linked it on the organization profile.

**Take this checklist as orientation for yourself, if this PR is ready for Maintainer Review**
- [x] My suggestion follows the [governance rules](https://github.com/openMSL/governance-and-documentation).
- [x] All commits of this PR are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
- [x] My changes generate no errors when passing CI tests. 
- [x] I updated all documentation (readmes incl. figures) according to my changes.
- [x] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

### CODE REVIEW

File: doc/related_work.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+It is used to test simulation models in a co-simulation directly in the GitHub CI pipeline.
+
+[ASAM OpenScenario](https://www.asam.net/standards/detail/openscenario/):
+How objects move in a scene is described in the OpenScenario format.
+It is used to test simulation models in a co-simulation directly in the GitHub CI pipeline.
+
+[ASAM OpenSimulationInterface](https://github.com/OpenSimulationInterface/open-simulation-interface):
+All simulation models on OpenMSL use OSI as the standard interface for model input and model output.
+
+[Modelica FMI](https://fmi-standard.org/):
```
Comment (jdsika) [inline doc/related_work.md:16]: Is OSMP officially a part of ASAM OSI or should it be mentioned here as well as FMI "template"?

File: doc/related_work.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+To specify a system for co-simulation, the SSP standard is used.
+This XML standard defines, which FMUs belong to a certain system and how they are connected in a so-called System Structure Definition (SSD).
+Also, parameters can be assigned to the different FMUs.
+
+## Other Projects
+
+[esmini](https://github.com/esmini/esmini):
+For co-simulation of OpenMSL models directly in the GitHub CI pipeline, esmini is used to play OpenScenario files as input for the models.
+
+[FMPy](https://github.com/CATIA-Systems/FMPy):
```
Comment (jdsika) [inline doc/related_work.md:30]: The official FMI checker is missing?

File: doc/related_work.md (hunk 1 above)
Comment (ClemensLinnhoff) [reply doc/related_work.md:16]: I added OSMP to the tools.

File: doc/related_work.md (hunk 2 above)
Comment (ClemensLinnhoff) [reply doc/related_work.md:30]: Done.

File: profile/README.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ The [ENVITED Research Cluster](https://envited.market/) is an initiative to crea
 Virtual validation is becoming an essential part of development processes for highly automated driving (HAD) functions. Standards for model and system interchange are vital for cross-company and cross-domain virtual
 integration and simulation of HAD functions.
 
+Therefore, OpenMSL is heavily based on various international standards and widely used open-source tools. A list of the utilized standards and tools can be found [here](../doc/related_work.md).
```
[the remark this answers is not in the document]
Comment (ClemensLinnhoff) [reply profile/README.md]: Done.

[the remark this answers is not in the document]
Comment (jdsika) [reply profile/README.md]: "Chief Sales Officer" Clemens Linnhoff -love it!

**Review by jdsika — APPROVED** [review]

**Pull request merged**